target_stitch/__init__.py

```diff
@@ -194,7 +271,12 @@ def persist_lines(stitchclient, lines):
             state = message.value
 
         elif isinstance(message, singer.SchemaMessage):
-            schemas[message.stream] = message.schema
+
+            # Draft4Validator will fail unless both the multipleOf in the
+            # schema and the value in the data are Decimal. So we need to
+            # convert all instances of multipleOf in the schema to
+            # Decimal.
+            schemas[message.stream] = ensure_multipleof_is_decimal(message.schema)
```

Comment:
What's the purpose of this? Further validation on the value of the `multipleOf ` key?